sdk/lib/crt/printf/streamout.c

```diff
@@ -432,6 +432,7 @@ streamout(FILE *stream, const TCHAR *format, va_list argptr)
             else if (chr == _T('w')) flags |= FLAG_WIDECHAR;
             else if (chr == _T('L')) flags |= 0; // FIXME: long double
             else if (chr == _T('F')) flags |= 0; // FIXME: what is that?
+            else if (chr == _T('z')) flags |= FLAG_INTPTR;
```

Comment:
Should this be supported here?
Is this file not synced with wine?

Reply:
> Should this be supported here?

WinXP/2k3 doesn't support this. But I think it is useful for interoperability.

>Is this file not synced with wine?

File `media/doc/WINESYNC.txt`:
```txt
msvcrt -
  sdk/lib/crt/conio/cputs.c             # Synced to WineStaging-1.9.16
  sdk/lib/crt/except/cpp.c              # Synced at 20080528
  sdk/lib/crt/except/cppexcept.c        # Synced at 20071111
  sdk/lib/crt/process/_cwait.c          # Synced to WineStaging-1.7.37
  sdk/lib/crt/signal/xcptinfo.c         # Synced to WineStaging-1.7.37
  sdk/lib/crt/string/scanf.c/h          # Synced to Wine-1.7.17
  sdk/lib/crt/string/strtoi64.c         # Synced to WineStaging-1.9.9
  sdk/lib/crt/string/strtok.c           # Synced to WineStaging-1.9.16
  sdk/lib/crt/string/strtok_s.c         # Synced to WineStaging-1.9.16
  sdk/lib/crt/string/strtoul.c          # Synced to WineStaging-1.9.9
  sdk/lib/crt/string/strxfrm.c          # Synced to Wine-1.9.16
  sdk/lib/crt/string/wcs.c              # Synced at 20080611
  sdk/lib/crt/string/wctype.c           # Synced to WineStaging-1.9.16
  sdk/lib/crt/string/wtoi64.c           # Synced to Wine-1.9.16
  sdk/lib/crt/mbstring/ischira.c        # Synced to Wine-1.9.16
  sdk/lib/crt/wine/heap.c               # Synced at 20080529
  sdk/lib/crt/wine/undname.c            # Synced to WineStaging-1.9.16
  sdk/lib/crt/process/thread.c          # Synced to WineStaging-1.7.55
```

No, not synced.